src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1166,13 +1166,9 @@
     <value>Name</value>
     <comment>Header for a control to determine the name of the profile. This is a text box.</comment>
   </data>
-  <data name="Profile_AcrylicHeader.Text" xml:space="preserve">
-    <value>Acrylic</value>
-    <comment>Header for a group of settings related to the acrylic texture rendering on the background of the app.</comment>
-  </data>
   <data name="Profile_TransparencyHeader.Text" xml:space="preserve">
-    <value>Transparency</value>
-    <comment>Header for a group of settings related to transparency, including the acrylic texture rendering on the background of the app.</comment>
+    <value>Translucency</value>

```

Comment:
This one I'm less sold on, cause now we've got vintage transparency too, which _is_ totally transparent.